openapi.yaml

```diff
@@ -198,7 +198,7 @@ paths:
                 $ref: '#/components/schemas/Error'
   '/pilotLog/{memberId}':
     get:
-      summary: '[VV, SYSADMIN, SELF] Pilot log entry of a specific account'
+      summary: '[SELF] Pilot log entry of a specific Member''s Pilotlog'
```

Comment:
@TheFail0r is das so vom PO gewollt?
Warum sollte das nicht auch von VV, SYSADMIN und FLUGWART eingesehen werden können?

Reply:
Sollte auf jeden nochmal erfragt werden.
Hatte dazu ja schon was in meinem Branch geschrieben.
Sollten da mal den PO nochmal fragen und dann anpassen.

Reply:
Es ist nirgends festgehalten, wer die sehen können muss. Soweit ich das überschauen kann, nicht mal der Member selbst. Sinn macht das halt nicht wirklich.

Reply:
Im Rollendokument steht nur eig. Flüge erfassen...
Das habe ich mal als PilotLogEntry angesehen und da steht halt nur aktives Mitglied.

Reply:
Ich würd sagen @TheFail0r regelt das mit'm PO, @twizzz und ich ragen da zu viel rum